### ISSUE TITLE
Cocoa pods spec has not been updated

### ISSUE THREAD

**Issue by epankratov** [ReactiveCocoa/ReactiveObjCBridge#59]
Hello. The released version contains outdated podspec files, as far as CocoaPods do not know anything about version 5.0.0: the master repo still refers to 4.0.0 version. It would be great to have both up-to-date

**Comment by mdiep** [comment]
PRs welcome! 🙂

**Comment by robertmusialek** [comment]
@mdiep I think PR with changes to pod spec has already been merged, but it was after 5.0.0 release had been created.
I think creating new release that includes commit with pod spec update is required.

**Comment by ikesyo** [comment]
It's on trunk now:

```
ReactiveObjCBridge
    - Versions:
      - 1.0.0 (2017-01-21 15:36:50 UTC)
      - 1.0.1 (2017-04-04 08:37:14 UTC)
      - 1.0.2 (2017-04-07 04:30:23 UTC)
      - 1.1.0 (2017-07-26 17:10:54 UTC)
      - 2.0.0 (2017-08-22 05:10:27 UTC)
      - 3.0.0 (2018-01-17 20:39:54 UTC)
      - 3.1.0 (2018-07-03 13:53:45 UTC)
      - 4.0.0 (2018-09-01 04:09:57 UTC)
      - 5.0.0 (2019-09-06 15:41:10 UTC)
    - Owners:
      - Syo Ikeda <suicaicoca@gmail.com>
      - Adam Sharp <adam@sharplet.me>
      - Anders Ha <anders@andersha.com>
```
